opt_einsum/backends/torch.py

```diff
@@ -41,14 +42,31 @@ def transpose(a, axes):
     return a.permute(*axes)
 
 
+@lru_cache(None)
+def get_einsum():
+    torch, _ = _get_torch_and_device()
+    if hasattr(torch, "_VF") and hasattr(torch._VF, "einsum"):
+        print("returning VF")
```

Comment:
If we remove the prints, I think we're good to go1

Reply:
Are the current test cases sufficient to verify this doesn't break anything?